dav-controller/api/templates/verified_credentials.html

```diff
@@ -273,292 +134,406 @@
+          <div v-if="state.current === 'intro'" class="d-flex flex-row justify-content-center">
+            <div class="header-desc intro my-2">
+              <div class="qr-code-image">
+                {{add_asset("hand-qrcode.svg")}}
+              </div>
+              <div class="text-start mx-2">
+                Scanning this QR code will send you a request to share your
+                information
               </div>
             </div>
-          </button>
-          <div class="scanned-mask">
-            <div class="message">QR code scanned</div>
           </div>
-          <div class="border">{{add_asset("dashed-border.svg")}}</div>
-          <img
-            src="data:image/jpeg;base64,{{image_contents}}"
-            alt="{{image_contents}}"
-            width="300px"
-            height="300px"
-          />
-        </div>
-        <div class="revealed-attribs" id="revealed-attribs"></div>
-        <div>
-          <button id="deep-link-btn" class="deep-link-button" title="Open in BC Wallet">
-            <div class="button-content">
-              <div class="fake-button">
-                <div class="icon">{{add_asset("mobile-wallet.svg")}}</div>
-                <div class="description">Open in BC Wallet</div>
+          <div v-if="state.current === 'success' && state.imageUrl">
+            <img
+              v-bind:src="state.imageUrl"
+              alt="Verified Picture"
+              style="flex-shrink: 0"
+              width="300px"
+            />
+          </div>
+          <div v-else class="media qr-code">
+            <button
+              id="refresh-button"
+              class="qr-button"
+              title="Refresh QR Code"
+              v-if="state.showRefresh"
+              v-on:click="reload"
+            >
+              <div
+                class="button-content btn fw-bolder btn-outline-primary bg-white fs-6 d-flex flex-row"
+              >
+                <div class="icon mx-1">{{add_asset("refresh.svg")}}</div>
+                <div class="description mx-1">Refresh QR code</div>
               </div>
+            </button>
+            <div v-if="state.showScanned" class="scanned-mask">
+              <div class="message">QR code scanned</div>
             </div>
-          </button>
+            <img
+              class="qr-code"
+              :class="{pending: state.current === 'pending'}"
+              src="data:image/jpeg;base64,{{image_contents}}"
+              alt="{{image_contents}}"
+              width="300px"
+            />
+          </div>
         </div>
-        <div>
-          <button id="new-request-btn" class="new-request-button" title="Create New Request">
-            <div class="button-content">
-              <div class="fake-button">
-                <div class="icon">{{add_asset("refresh.svg")}}</div>
-                <div class="description">Create New Request</div>
-              </div>
-            </div>
-          </button>
```

Comment:
This got removed, but we should re-introduce it: we want users to be able to stay on the same page and go through verifications without leaving. Maybe use `Start New Age Verification` rather than request, but functionality is the same - refresh the page and start from scratch.